packages/gatsby-source-wordpress/src/steps/create-schema-customization/index.js

```diff
@@ -86,9 +91,10 @@ const customizeSchema = async ({ actions, schema, store: gatsbyStore }) => {
       fields: nonNodeRootFields,
       interfaces: [`Node`],
     },
-    isAGatsbyNode: true,
   })
 
+  typeDefs.push(wpType)
```

Comment:
this is an inconsequential change, just put the .push closer to what's being pushed